core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -241,16 +242,13 @@ public void commitBlock(long sessionId, long blockId, boolean pinOnCreate)
       throws IOException {
     LOG.debug("commitBlock: sessionId={}, blockId={}, pinOnCreate={}",
         sessionId, blockId, pinOnCreate);
-    long lockId = mLockManager.lockBlock(sessionId, blockId, BlockLockType.WRITE);
-    try {
+    try (Closeable lock = mLockManager.acquireBlockLock(sessionId, blockId, BlockLockType.WRITE)) {
```

Comment:
assume we can apply this pattern to `removeBlockInternal` and others in class too, right?

Reply:
Yes, I've double checked that,  but I would like to split the change into multiple commits, then it would easier to catch the deadlock or regression.